jsapp/js/components/processing/singleProcessingStore.ts

```diff
@@ -173,20 +162,25 @@ class SingleProcessingStore extends Reflux.Store {
     this.data.translations = [];
     this.data.translationDraft = undefined;
     this.data.source = undefined;
-    this.data.activeTab = SingleProcessingTabs.Transcript;
     this.data.isPristine = true;
   }
 
   public get currentAssetUid() {
-    return getSingleProcessingRouteParameters().uid;
+    // To avoid too much complexity in the code, we pass empty string instead
+    // of undefined
+    return getProcessingRouteParts().assetUid || '';
```

Comment:
@LMNTL I don't like that I had to put `|| ''` here (and below), but I feel like this is better than returning `string | undefined` and adding many `if`s through the code. WYT?

Reply:
Empty strings work just fine, good idea 👍